src/pages/reservations/Resvervations.js

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+
+import { reservationsThunk } from '../../slices/reservationSlice/reservationSlice';
+import { setAlert } from '../../slices/appSlice/appSlice';
+import ReservationCard from './ReservationCard';
+
+export default function Reservations() {
+  const dispatch = useDispatch();
+  const token = useSelector((state) => state.auth.token);
+  const reservations = useSelector((state) => state.reservation.reservations);
+
+  useEffect(() => {
+    dispatch(reservationsThunk(token)).then((res) => {
+      if (res.error) {
+        dispatch(setAlert(res.payload));
+      }
+    });
+  }, [dispatch, token]);
+
+  return (
+    <div>
+      <h1 className="reservations_title">My Reservations</h1>
+      <div className="reservation_container">
+        {reservations.map((data) => (
+          <ReservationCard
+            key={data.reservation.id}
+            id={data.reservation.id}
+            startTime={data.reservation.start_time}
+            endTime={data.reservation.end_time}
+            duration={data.reservation.duration}
+            imageUrl={data.image_url}
+          />
+        ))}
+      </div>
+    </div>
```

Comment:
Can you add some loading logic like if the API is fetching instead of showing blank screen instead show some "Loading..." text or some nice log and animations are fine too.

Reply:
Please handle the margins that is being outside.
![Screenshot from 2023-04-06 21-53-57](https://user-images.githubusercontent.com/105173981/230425457-6b9578b3-dc60-4be7-8666-16db7d30c27f.png)